libs/packages/sam-formly/src/lib/formly-reset/formly-reset.component.spec.ts

```diff
@@ -61,40 +61,6 @@ describe('SDS Formly Reset', () => {
     expect(testComp.model).toEqual({ flat: null, nested: { nestedInput: null } });
   });
 
-  it("should reset fields with defaultValue to null if value isn't set in model when clicked", () => {
```

Comment:
This test was removed as the behavior being tested for was removed in @ngx-formly/core@5.10.10. As shown by https://github.com/ngx-formly/ngx-formly/issues/2621, the behavior expected by the test was not the intended behavior and will not the behavior going forward. As such, I have removed this test